DOCUMENTATION.md

```diff
@@ -92,42 +92,77 @@ The `nav` attribute describes an array of navigation AsciiDoc files uses to buil
+
+Lets say we need to add an attribute that points to our JIRA issue for issue links, that would be an example of an attrbute that would be defined in the playbook.  But if we needed to add an attribute that points to a specific version of a naven artifact or reference a specific version of a dependency, that's more appropriate for the `antora.yml` component descriptor located in this repository. 
 
-The current attributes file looks similar to the following:
+The current `antora.yml` component descriptor looks similar to the following:
+```
+asciidoc:
+  attributes:
+    debezium-version: '1.1.0.Final'
+    debezium-dev-version: '1.2'
+    debezium-kafka-version: '2.4.0'
+    debezium-docker-label: '1.1'
+    install-version: '1.1'
+    assemblies: '../assemblies'
+    modules: '../../modules'
+    mysql-connector-plugin-download: 'https://repo1.maven.org/maven2/io/debezium/debezium-connector-mysql/1.1.0.Final/debezium-connector-mysql-1.1.0.Final-plugin.tar.gz'
+    mysql-version: '8.0'
+    confluent-platform-version: '5.3.1'
+    strimzi-version: '0.13.0'
 ```
-:moduledir: ..
-:attachmentsdir: {moduledir}/assets/attachments
-:examplesdir: {moduledir}/examples
-:imagesdir: {moduledir}/assets/images
-:partialsdir: {moduledir}/pages/_partials
 
-:debezium-version: 0.10.0.Beta4
-:debezium-dev-version: 0.10
-:debezium-kafka-version: 2.3.0
-:debezium-docker-label: 0.10
-:install-version: 0.10
-:confluent-platform-version: 5.1.2
-:strimzi-version: 0.10.0
+Attributes are defined by creating a nested yaml structure under `asciidoc.attributes` where the key-value attribute pairs are to be defined.  
 
+The playbook files in the website repository use the same layout, shown here:
 ```
+# Global asciidoc attributes here, used across all versions of documentation
+asciidoc:
+  attributes:
+    prodname: 'Debezium'
+    context: 'debezium'
+    jira-url: 'https://issues.redhat.com'
+    # because of how handlebars templates work with page.attributes, this must be prefixed with "page-"
+    page-copyright-year: '2020'
+``` 
 
-## What to change _after_ releasing new version
+**NOTE**: Given that the Debezium documentation is consumed downstream by other processes, do not define attributes in the `_attributes.adoc` file and use it as an include nor should you define attributes locally in a given .adoc file.
 
-It's important that 2 files be properly maintained after the repository transitions between releases
+## What to change _before_ releasing a new version
 
-* _attributes.adoc
-* antora.yml
+The `antora.yml` file in the `master` branch always uses version called _master_.  It is important right before a release when applying the change log and contributor changes to modify this file to reference the correct major and minor version.  It's also important to make any necessary changes to version-specific attributes in this file at the same time.
```

Comment:
Same as above: can you change to one thought/sentence per line, avoiding super-long lines? Thanks!